requirements.txt

```diff
@@ -1 +1,10 @@
-
+beautifulsoup4==4.10.0
+ghapi==0.1.19
+matplotlib==3.3.4
```

Comment:
do not mix test and CI dependencies with development ones. leave only your dependencies

Reply:
Ok